tests/matchers/test_base.py

```diff
@@ -0,0 +1,36 @@
+from traccuracy._tracking_graph import TrackingGraph
+from traccuracy.matchers._base import Matched, Matcher
+
+from tests.test_utils import get_movie_with_graph
+
+
+class DummyMatcher(Matcher):
+    def _compute_mapping(
+        self, gt_graph: TrackingGraph, pred_graph: TrackingGraph
+    ) -> Matched:
+        return Matched(gt_graph, pred_graph, [])
+
+
+class DummyMatcherParam(DummyMatcher):
+    def __init__(self, param="default"):
+        self.param = param
+
+
+def test_matched_info():
+    matcher = DummyMatcher()
+    # Check that matcher info is correctly generated
+    assert matcher.info["name"] == "DummyMatcher"
+    # Only one key because there are no parameters
+    assert len(matcher.info.keys()) == 1
+
+    matcher = DummyMatcherParam()
+    # Check for parameter key
+    assert "param" in matcher.info
```

Comment:
Also test that the default value is present? And/or that if you initialize with not the default, it changes?